includes/block-patterns/lesson/templates/default-with-quiz.php

```diff
@@ -0,0 +1,41 @@
+<?php
+/**
+ * Default with Quiz lesson pattern content.
+ *
+ * @package sensei-lms
+ */
+
+if ( ! defined( 'ABSPATH' ) ) {
+	exit; // Exit if accessed directly.
+}
+
+?>
+<!-- wp:paragraph {"placeholder":"<?php esc_html_e( 'Write lesson content...', 'sensei-lms' ); ?>"} -->
+<p></p>
+<!-- /wp:paragraph -->
+
+<!-- wp:sensei-lms/quiz {"options":{"passRequired":false,"quizPassmark":0,"autoGrade":false,"allowRetakes":false,"showQuestions":null,"randomQuestionOrder":false,"failedIndicateIncorrect":true,"failedShowCorrectAnswers":true,"failedShowAnswerFeedback":true,"buttonTextColor":null,"buttonBackgroundColor":null,"pagination":{"paginationNumber":null,"showProgressBar":false,"progressBarRadius":6,"progressBarHeight":12,"progressBarColor":null,"progressBarBackground":null},"enableQuizTimer":"1","timerValue":300},"isPostTemplate":true} -->
```

Comment:
I tried to keep everything "as it is" when copying from the sensei-demo site, to avoid future issues. I am a little concerned that we have to update or add new patterns in the future and we start making inconsistencies between the existing and the new ones...
Anyway I am going to use your suggestion on the quiz block.

Regarding the quiz at the end. I did it like that, but then moved the Lesson Actions block to the end after [this comment](p1654779781166499/1654710581.646339-slack-C8QJAM52B) from Ronnie.
My initial reasoning was the same as yours, but I asked and after Ronnie's comment I changed it to be the Lesson Actions the last element.